includes/writer.js

```diff
@@ -47,6 +52,19 @@ module.exports = function(couchDbUrl, bufferSize, parallelism) {
       }
       cb();
     });
+
+    response.on('response', function(resp) {
+      if (resp.statusCode === 401) {
```

Comment:
Ideally we should do the following:
- Distinguish between permanent (4xx) and transient (5xx) codes. The latter can be re-tried as and when we have re-try logic in place.
- Have a shared lookup table which translates HTTP status codes into human readable errors which make sense in the context of the tool. I think there is a lot of value in a message like `Database ${couchDbUrl} does not have correct permissions. Check that you have the correct permissions before restoring` rather than `Error 401`.